packages/ses/src/lockdown-shim.js

```diff
@@ -124,16 +112,22 @@ export function lockdown(options = {}) {
   repairLegacyAccessors();
 
   /**
-   * 4. HARDEN to share the intrinsics.
+   * 3. HARDEN to share the intrinsics.
    */
 
   // Circumvent the override mistake.
   const detachedProperties = enablePropertyOverrides(intrinsics);
 
   // Finally register and optionally freeze all the intrinsics. This
   // must be the operation that modifies the intrinsics.
-  harden(intrinsics, registerOnly);
-  harden(detachedProperties, registerOnly);
+  lockdownHarden(intrinsics, registerOnly);
```

Comment:
Where is this `harden` (aka `lockdownHarden`) function that takes a second parameter? I couldn't find it.

Reply:
https://github.com/Agoric/SES-shim/blob/0e83a286dc01ee16d20434548bd8bac0faab30bc/packages/ses/src/lockdown-shim.js#L42

Reply:
Whose source is at https://github.com/Agoric/SES-shim/blob/0e83a286dc01ee16d20434548bd8bac0faab30bc/packages/make-hardener/src/main.js#L67 which takes only one parameter?

Reply:
I checked. `lockdownHarden`, which is just the local name for `harden` that is usable before `lockdown` finishes, does not accept a second argument. To me, this looks like the `registerOnly` behavior does not work and (because tests pass) is not tested. I don’t know what it is meant to do.

Reply:
What is `registerOnly` supposed to mean or do? What is it needed for?

Reply:
There is no indication anywhere in the code or documentation of what `registerOnly` means or should do. I would be glad to erase it in this change.

Reply:
Yes, please do.

For a small security kernel, I'd prefer to err on the side of removing things we don't know the purpose of. Sometimes, in review, I'll know and clarify.